Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1772,6 +1774,8 @@ def replace_str(src_str):
            r'(?:(?:\/|\?)[-\w\d+&@#\/%=~_$?!\-:,.\(\);]*[\w\d+&@#\/%=~_$\(\);])?'
 cveRegex = r'(?i)^cve-\d{4}-([1-9]\d{4,}|\d{4})$'
 
+domainGlobRegEx = r'\b(\*\.)([\w\d]+\.)+[\w\d]+\b'
```

Comment:
Change the variable name to use snake-case, please. I don't think this regex is sufficient as is. If you'd like, I don't mind helping with that.

Reply:
The variable name is as it is because all of the rest are like that. 
Why do you think it is insufficient?